pyrasterframes/src/main/python/docs/raster-write.pymd

```diff
@@ -85,6 +84,19 @@ If there are many _tile_ or projected raster columns in the DataFrame, the GeoTI
 os.remove(outfile)
 ```
 
+## Overview Rasters
+
+In cases where writing and reading to/from a GeoTIFF isn't convenient, RasterFrames provides the `rf_agg_overview_raster` aggregate function, where you can construct a single raster (rendered as a tile) downsampled from all or a subset of the dataframe. This allows you to effectively construct the same operations the GeoTIFF writer performs, but without the file I/O.
+
+Because a Dataframe may contain data with varying CRSs, and the rendered raster needs to have a single CRS, an "Area of Interest" (AOI) is required in a predetermined CRS. In the case of `rf_agg_reprojected_extent`, the AOI needs to be in commonly used ["web mercator"](https://en.wikipedia.org/wiki/Web_Mercator_projection) CRS. 
+
+```python, overview
+from pyrasterframes.rf_types import Extent
+target = spark_df.withColumn('extent', rf_extent('proj_raster')).withColumn('crs', rf_crs('proj_raster')) 
+aoi = Extent.from_row(target.select(rf_agg_reprojected_extent('extent', 'crs', 'EPSG:3857')).first()[0])
+target.select(rf_agg_overview_raster(rf_tile('proj_raster'), 512, 512, aoi, 'extent', 'crs')).first()[0]
```

Comment:
Can we leverage the `aoi`'s aspect ratio to help set the number of rows ?  Maybe...


```suggestion
aspect = aoi.width() / aoi.height()
target.select(rf_agg_overview_raster(rf_tile('proj_raster'), 512 * aspect, 512, aoi, 'extent', 'crs')).first()[0]
```